.github/workflows/doxygen.yml

```diff
@@ -17,20 +17,29 @@ jobs:
 
       - name: Create documentation branch
         run: git checkout -b gh-pages
-        
+
       - name: Prepare Secrets
         run: echo "${{ secrets.GITHUB_TOKEN }}" >> $GITHUB_WORKSPACE/secret.txt
 
       #Mandatory step it appears otherwise pushing the commit will fail
-      #Switched username and e-mail to something identifiable?
       - name: Configure Git
         run: |
           gh auth login --with-token < $GITHUB_WORKSPACE/secret.txt
           git config user.name "Novel-chan"
           git config user.email "admin@novelrt.dev"
 
+      - name: Prepare build environment
+        run: |
+          pip3 install conan
+          conan config install https://github.com/NovelRT/ConanConfig.git
+
+      - name: Generate Build System
+        run: conan install . -if build --build=missing --profile linux-clang10-amd64-RelWithDebInfo
+
       - name: Generate Documentation
-        run: scripts/cibuild.sh --documentation -DDOXYGEN_OUTPUT_DIRECTORY=$GITHUB_WORKSPACE/docs
+        run: |
+          cmake -S . -B build -DCMAKE_BUILD_TYPE="RelWithDebInfo" -DDOXYGEN_OUTPUT_DIRECTORY=$GITHUB_WORKSPACE/docs
+          cmake --build build --config RelWithDebInfo --target Doxygen
```

Comment:
There's no easy way to get Conan to just run target doxy, so I spelled out the full commands here.